packages/appstore/src/components/modals/wallet-modal/wallet-modal.tsx

```diff
@@ -1,30 +1,57 @@
-import React, { useEffect } from 'react';
+import React, { useEffect, useMemo } from 'react';
 import { Modal, Loading } from '@deriv/components';
+import { useActiveWallet } from '@deriv/hooks';
+import { routes } from '@deriv/shared';
+import { observer, useStore } from '@deriv/stores';
 import WalletModalHeader from './wallet-modal-header';
 import WalletModalBody from './wallet-modal-body';
-import { observer, useStore } from '@deriv/stores';
-import { useActiveWallet } from '@deriv/hooks';
 
 const WalletModal = observer(() => {
     const store = useStore();
-
+    const wallet = useActiveWallet();
     const {
         client: { balance, currency, landing_company_shortcode: shortcode, is_authorize, switchAccount },
         ui: { is_dark_mode_on, is_wallet_modal_visible, is_mobile, setIsWalletModalVisible },
-        traders_hub: { active_modal_tab, active_modal_wallet_id, setWalletModalActiveTab },
+        traders_hub: {
+            active_modal_tab,
+            active_modal_wallet_id,
+            setWalletModalActiveTab,
+            setWalletModalActiveWalletID,
+        },
     } = store;
```

Comment:
@heorhi-deriv  Just a nitpick, you can ignore it, but in my opinion we don't need to create a separate `const store = useStore()` and use it only in one place.
```suggestion
    } = useStore();
```